src/math2d/line.rs

```diff
@@ -0,0 +1,30 @@
+use super::point::Point;
+
+#[derive(Debug, Clone, Copy, PartialEq, PartialOrd)]
+pub struct Line {
+    start: Point,
+    end: Point,
+}
+
+impl Line {
+    pub fn new(start: &Point, end: &Point) -> Self {
+        Line {
+            start: *start,
+            end: *end,
+        }
+    }
+
+    pub fn get_start(&self) -> &Point {
+        &self.start
```

Comment:
Also I would return these by value. The Point struct is of size 64 bit which fits in a register

Reply:
I changed it to return by value, but since I made Point's members private, I'm return not the Point struct itself, but a (f32, f32) tuple with the point's coordinates.